README.md

```diff
@@ -1,12 +1,81 @@
 
 This is the API used for Email orchestration between CLA services and GOV.UK Notify.
 
+## Development in a virtual environment
+
+#### 1) Clone the repository
+
+```
+git clone https://github.com/ministryofjustice/laa_govuk_notify_orchestrator.git
+```
+
+#### 2) Ensure you have the correct Python3 version (3.11)
+
+```
+python3 --version
+```
+
+If this is below 3.11 please ensure you have installed python3.11 somewhere on your system.
+
+#### 3) Setup and activate a Python virtual environment
+
+```
+python3.11 -m venv env --prompt=\(notify_api\)
+
+source env/bin/activate
+```
+
+##### 3.1) Ensure you are running the correct version of Python in your virtual environment
+
+```
+python --version
+
+pip --version
+```
+
+Ensure both state 'Python 3.11'
+
+#### 4) Update pip and install the required Python dependancies
+
+```
+pip install -U pip
+
+pip install -r requirements.txt
+```
+
+#### 5) Launch the service
+
+```
+python start.py
```

Comment:
No such file or directory :)

Reply:
This should now be fixed, this was renamed to `python manage.py`.

Many thanks! 👍